lti_consumer/tests/unit/test_lti.py

```diff
@@ -129,30 +129,27 @@ def setUp(self):
         super(TestLtiConsumer, self).setUp()
         self.lti_consumer = LtiConsumer(self.xblock)
 
-    @patch(
-        'lti_consumer.lti.get_oauth_request_signature',
-        Mock(return_value=(
-            'OAuth oauth_nonce="fake_nonce", '
-            'oauth_timestamp="fake_timestamp", oauth_version="fake_version", oauth_signature_method="fake_method", '
-            'oauth_consumer_key="fake_consumer_key", oauth_signature="fake_signature"'
-        ))
-    )
-    @patch(
-        'lti_consumer.lti_consumer.LtiConsumerXBlock.prefixed_custom_parameters',
-        PropertyMock(return_value={u'custom_param_1': 'custom1', u'custom_param_2': 'custom2'})
-    )
-    @patch(
-        'lti_consumer.lti_consumer.LtiConsumerXBlock.lti_provider_key_secret',
-        PropertyMock(return_value=('t', 's'))
-    )
-    @patch('lti_consumer.lti_consumer.LtiConsumerXBlock.user_id', PropertyMock(return_value=FAKE_USER_ID))
```

Comment:
Could we keep this patching where it was? The `patch_signed_parameters` decorator is elegant, but it obscures what's going on and moves it further away from the test itself.

Reply:
I found that copying the `@patch` statements would be redundant, so I thought DRY is better.

Reply:
Thanks, I'll keep it as-is for now.